libs/api/domains/content-search/src/lib/contentSearch.service.ts

```diff
@@ -20,15 +21,15 @@ export class ContentSearchService {
     return SearchIndexes[languageCode] ?? SearchIndexes.is
   }
 
-  mapFindAggregations(aggregations): TagCount[] {
-    if (!aggregations) {
+  mapFindAggregations(aggregations: TagAggregationResponse): TagCount[] {
```

Comment:
can you add a test to guard against this bug happening again?

Reply:
Yes adding test here is on the roadmap